### ISSUE TITLE
v1.4.1 build error

### ISSUE THREAD

**Issue by lcgogo** [PlatONnetwork/PlatON-Go#2159]
v1.4.1 make platon 报错，用的ubuntu20.04 go 1.17
v1.4.0 没有问题
感觉是 p2p/util.go 里面的一个函数报错
```
cli@172-16-3-109:~/PlatON-Go/p2p$ go version
go version go1.17.13 linux/amd64
cli@172-16-3-109:~/PlatON-Go/p2p$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
```

```shell
make platon
....
golang.org/x/text/internal/colltab
github.com/fatih/color
github.com/dop251/goja/parser
github.com/dlclark/regexp2
github.com/PlatONnetwork/PlatON-Go/internal/jsre/deps
github.com/PlatONnetwork/PlatON-Go/internal/web3ext
github.com/PlatONnetwork/PlatON-Go/ethclient
golang.org/x/text/collate
github.com/naoina/go-stringutil
github.com/naoina/toml/ast
github.com/naoina/toml
github.com/dop251/goja
github.com/PlatONnetwork/PlatON-Go/accounts/usbwallet
github.com/PlatONnetwork/PlatON-Go/internal/jsre
util.go:47: exit status 2
exit status 1
make: *** [Makefile:18: platon] Error 1
```

**Comment by dolphintwo** [comment]
https://github.com/PlatONnetwork/PlatON-Go/commit/a36683d2260f50f22cfc7bffbf2c16e3f3a7d5cd#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R82
中module `github.com/rivo/uniseg` 要求go 1.18+
参: https://github.com/rivo/uniseg/blob/3628fa14e3cd43899a3277293c4049e42511b597/go.mod#L3

**Comment by lcgogo** [comment]
> [a36683d#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R82](https://github.com/PlatONnetwork/PlatON-Go/commit/a36683d2260f50f22cfc7bffbf2c16e3f3a7d5cd#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R82) 中module `github.com/rivo/uniseg` 要求go 1.18+ 参: https://github.com/rivo/uniseg/blob/3628fa14e3cd43899a3277293c4049e42511b597/go.mod#L3

Thanks, can build under go 1.18

Please update 

```
The requirements to build PlatON-Go are:

    OS:Windows10/Ubuntu18.04
    [Golang](https://golang.org/doc/install) :version 1.17+
    [cmake](https://cmake.org/) :version 3.0+
    [g++&gcc](http://gcc.gnu.org/) :version 7.4.0+
```

in the repo README.md

**Comment by dolphintwo** [comment]
Also update in [`go.mod`](https://github.com/PlatONnetwork/PlatON-Go/blob/develop/go.mod#L3)

**Comment by benbaley** [comment]
Thanks :)

**Comment by benbaley** [comment]
fixed in #2196

**Comment by benbaley** [comment]
close by done
